src/worker.cc

```diff
@@ -203,6 +205,26 @@ void worker(
                         }
                         maybeConstituents =
                             Constituents(constituents, namedConstituents);
+                    } else if (args.applyExpr != "") {
```

Comment:
Do we only run apply if it's not a derivation? Why not always do that?

Reply:
Ah. this is about constituents

Reply:
@Ma27 do we want to have apply also working if we do have constituents?

Reply:
I don't see any reason against it.

Reply:
I have no issue with having it aside of constituents, what do you think should the precedence be?
i.e. if `--apply` produces a `constituents` attribute, should that be overridden by `constituents`, or should we side step that question and put the result of `apply` into a separate attribute?